packages/slice-machine/components/Button/index.tsx

```diff
@@ -34,6 +35,7 @@ export const Button: React.FunctionComponent<ButtonProps> = ({
   onClick,
   sx = {},
   variant = "primary",
+  spinnerColor = "grey01",
```

Comment:
not sure this should be modified for consistency in the APP design, but i'm guessing you followed the design.

Reply:
It's more about the white buttons not being able to take a grey spinner since you can't see it. I agree that it's not great though and should ideally come from the button variant. Not sure that's possible though

Reply:
yeah that's what I understood too, we could make a function here to match variant with spinner color, would be ideal.
Not absolutely required for right now tho, it can wait.